### ISSUE TITLE
Search Button not working as expected

### ISSUE THREAD

**Issue by KSib** [a85/POSTMan-Chrome-Extension#3885]
## App Details:

```
Postman for Windows
Version 5.3.2
Windows 10 Enterprise 64-bit (Version: 1709. OS Build: 16299.64)
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: Encountered today
 2. Expected behaviour: When you click the Search icon after sending a request, it should only take one click to make the search menu appear, but it takes two for some reason.
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app): file:///C:/Users/[redacted]/AppData/Local/Postman/app-5.3.2/resources/app.asar/html/mode-auto.js net::ERR_FILE_NOT_FOUND
 4. Screenshots (if applicable):
![2017-11-30_10-25-37](https://user-images.githubusercontent.com/1589238/33441835-4375cacc-d5b9-11e7-8fd2-88ceda7038d4.gif)

1. Open postman.
2. Open the postman Echo folder
3. Select the DisgestAuth Request
4. Click Send
5. Click the search icon on the right
6. Click the X that appears
7. Click the Search icon again: nothing happens
8. Click the search icon one more time: the search box appears.

**Comment by sdnts** [comment]
@KSib I'm unable to reproduce this, does this happen for all requests in your case or just this one?

**Comment by KSib** [comment]
All requests

**Comment by KSib** [comment]
Happens on my home pc as well Windows 10 Pro 64bit

**Comment by deepakpathania-zz** [comment]
I was able to reproduce this on the linux app. On closing the search dialog by clicking on X, the dialog doesn't open up again on a single click of the search icon(requires two clicks to open). Works fine if closed by clicking on the search icon itself.

**Comment by bhargavkaranam** [comment]
This issue has been fixed in the latest version (6.5.x) of the Postman app.
